app/websocket_router.go

```diff
@@ -63,10 +63,10 @@ func (wr *WebSocketRouter) ServeWebSocket(conn *WebConn, r *model.WebSocketReque
 			conn.SetSessionToken(session.Token)
 			conn.UserId = session.UserId
 
-			wr.app.HubRegister(conn)
-
 			resp := model.NewWebSocketResponse(model.STATUS_OK, r.Seq, nil)
 			conn.Send <- resp
+
```

Comment:
Actually, I guess it's _possible_ for an event to be triggered in between these lines... maybe I should make the tests wait on the "hello" too.